### PR TITLE
add awesome-rl-for-cybersecurity

### PR DESCRIPTION
**Pull request by Limmen**

### CODE REVIEW

**Comment by cat998** [comment]
已收到！

**Comment by mokshdudeja** [comment]
Let's do coding with me I have made 2 games both are private in my git hub account

**Review by Kdpwgoodone — APPROVED** [review]

**Review by Rayhubi — APPROVED** [review]

**Comment by Rayhubi** [comment]
agreed

**Review by Technetium1 — APPROVED** [review]
This is good

**Comment by ronhUs-busrur-7tuxfo** [comment]
Thank you

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Repository | Description
 [AppSec](https://github.com/paragonie/awesome-appsec)								| Resources for learning about application security
 [Asset Discovery](https://github.com/redhuntlabs/Awesome-Asset-Discovery)    | List of resources which help during asset discovery phase of a security assessment engagement
 [Bug Bounty](https://github.com/djadmin/awesome-bug-bounty) 						| List of Bug Bounty Programs and write-ups from the Bug Bounty hunters
-[Capsulecorp Pentest](https://github.com/r3dy/capsulecorp-pentest) 						| Vagrant+Ansible virtual network penetration testing lab. Companion to "The Art of Network Penetration Testing" by Royce Davis 
+[Capsulecorp Pentest](https://github.com/r3dy/capsulecorp-pentest) 						| Vagrant+Ansible virtual network penetration testing lab. Companion to "The Art of Network Penetration Testing" by Royce Davis
```
Comment (ronhUs-busrur-7tuxfo) [inline README.md:17]: Okay let's do this

**Review by ronhUs-busrur-7tuxfo — COMMENTED** [review]
Thank you

**Pull request merged**